src/instructlab/taxonomy/diff.py

```diff
@@ -0,0 +1,83 @@
+# Standard
+import logging
+import os
+
+# Third Party
+from git import GitError
+import click
+import yaml
+
+# First Party
+from instructlab import helpers, utils
+from instructlab.data import generator
+
+
+@click.command()
+@click.option(
+    "--taxonomy-path",
+    type=click.Path(),
+    help=f"Path to {helpers.DEFAULT_TAXONOMY_REPO} clone or local file path.",
+)
+@click.option(
+    "--taxonomy-base",
+    help="Base git-ref to use for taxonomy.",
+)
+@click.option(
+    "--yaml-rules",
+    type=click.Path(),
+    default=None,
```

Comment:
This is different than what you did above

Reply:
copied from lab.py